course-definition.yml

```
@@ -61,6 +61,13 @@ extensions:
       Along the way, you'll learn about how capture groups and backreferences work.
       [1]: https://learn.microsoft.com/en-us/dotnet/standard/base-types/backreference-constructs-in-regular-expressions#numbered-backreferences
 
+  - slug: "filesearch"
```

```suggestion
  - slug: "file-search"
```

Consistent with other challenges